src/Page/NotAlone.elm

```diff
@@ -94,14 +85,29 @@ renderCard model journeyCardPosition =
         renderInitCard journeyCardPosition
 
 
-renderInitCard : JourneyCard -> List (Html msg)
+renderInitCard : JourneyCard -> List (Html Msg)
 renderInitCard journeyCardPosition =
     let
         journeyContent =
             journeyContentFromCardPosition journeyCardPosition
     in
     [ p [ css [ quoteStyle ] ] [ text (t journeyContent.relatable) ]
+    , div [ css [ closedStyle ] ]
+        [ p [ css [ quoteStyle ] ] [ text (t journeyContent.hopeful) ]
+        , p [ css [ quoteStyle ] ] [ text (t journeyContent.statement) ]
+        ]
     , p [ css [ detailsStyle ] ] [ text ("- " ++ t journeyContent.name ++ ", " ++ t journeyContent.age) ]
+    , verticalSpacing
+    , button [ css [ continueButtonStyle ], onClick (ToggleJourney journeyCardPosition) ]
+        [ span [ css [ continueTextStyle ] ] [ text (t ExpandQuoteButton) ]
+        ]
+    , div [ css [ closedStyle ] ]
+        [ p []
```

Comment:
This content needs to be here for the animation to work, right? I think yo explained it before.